styleguide/src/components/Cards/SimpleList.jsx.map

```diff
@@ -0,0 +1 @@
+{"version":3,"file":"SimpleList.jsx","sourceRoot":"","sources":["SimpleList.tsx"],"names":[],"mappings":"AAAA,OAAO,KAAK,MAAM,OAAO,CAAA;AACzB,OAAO,KAAK,MAAM,yBAAyB,CAAA;AAC3C,OAAO,SAAS,MAAM,6BAA6B,CAAA;AACnD,OAAO,IAAI,MAAM,wBAAwB,CAAA;AACzC,OAAO,SAAS,MAAM,SAAS,CAAA;AAO/B,MAAM,UAAU,GAAG,CAAC,EAAE,QAAQ,EAAE,KAAK,EAAU,EAAE,EAAE;IACjD,OAAO,CACL,CAAC,IAAI,CACH;MAAA,CAAC,SAAS,CAAC,CAAC,KAAK,CAAC,EAAE,SAAS,CAE7B;;MAAA,CAAC,KAAK,CAAC,KAAK,CAAC,CAAC,EAAE,WAAW,EAAE,OAAO,EAAE,CAAC,CACrC;QAAA,CAAC,SAAS,CAAC,CAAC,QAAQ,CAAC,EAAE,SAAS,CAClC;MAAA,EAAE,KAAK,CACT;IAAA,EAAE,IAAI,CAAC,CACR,CAAA;AACH,CAAC,CAAA;AAED,eAAe,UAAU,CAAA"}
```

Comment:
Should we be checking in `.map` files? This is [the only opinion I could find](https://stackoverflow.com/questions/26462852/typescript-which-files-should-i-check-in) after a quick search, and it doesn't seem very definitive. Also, I see this is in `src` and I'm wondering checking it in is actually necessary.

Reply:
I feel like we should commit them because it helps with navigation. Otherwise each dev would need to run `yarn workspace @chainlink/styleguide build` when they pull fresh changes.

Reply:
sorry for interjecting but hook it into a postinstall? 
ill take a review look tonight